static/js/components/FollowupRequestLists.jsx

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+
+import * as Actions from "../ducks/source";
+import styles from "./FollowupRequestLists.css";
+
+import EditFollowupRequestDialog from "./EditFollowupRequestDialog";
+
+const FollowupRequestLists = ({
```

Comment:
```suggestion
const FollowupRequestList = ({
```

Reply:
see above - want to keep this as is as multiple lists are rendered in general